homeassistant/components/tankerkoenig/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: No custom actions provided.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
```

Comment:
there are no data descriptions for the user step, as the form is already self explaining and IMO adding further text on this form would make it overwhelming and I don't think it would add value to it

![image](https://github.com/user-attachments/assets/acf3a109-3f39-487b-b501-65cd3363e318)

Reply:
with data descriptions it is a bit "full of text", but not as bad, as I thought before

![image](https://github.com/user-attachments/assets/c2fae15b-271f-4da5-9b4b-02fd570354e0)